src/app/components/challenge/challenge.component.html

```diff
@@ -125,7 +125,7 @@
                                     [routerLink]="['./leaderboard']"
                                     class="text-light-black fw-semibold fs-15">
                                     <i class="fa fa-th-list"></i> Leaderboard</a></li>
-                            <li *ngIf=""><a
+                            <li *ngIf="challenge.forumURL && challenge.enable_forum"><a
```

Comment:
Hey, I think it should be `challenge.forum_url` instead of `challenge.forumURL`.

Reply:
Done!! @Sanji515